generators/generator-botbuilder/generators/app/templates/core/package-with-tests.json.js

```diff
@@ -17,22 +17,22 @@
     },
     "dependencies": {
         "@microsoft/recognizers-text-data-types-timex-expression": "^1.1.4",
```

Comment:
Not a diff in the PR, but this is the only recognizers-text dependency that's unpinned in the generators.

Reply:
Resolved via https://github.com/microsoft/BotBuilder-Samples/pull/2411 🚀